config/settings/common.py

```diff
@@ -114,6 +114,11 @@
     }
 DATABASES["default"]["ATOMIC_REQUESTS"] = True
 
+# Turnstile
+TURNSTILE_SECRET = env(
+    "TURNSTILE_SECRET", default="1x0000000000000000000000000000000AA"
```

Comment:
Fine to merge as-is; I wonder in the future though, whether we should error with a sensible message if there's no secret in the `.env` file, rather than just falling silently to a nonsense default.

Reply:
The default fall-back is one of Cloudflare's testing keys, which always produces a pass. It means that if people are developing elsewhere they don't have to worry about putting in testing keys, and can just happily fall back to the default.

Reply:
So if we accidentally forgot to set the environment variable, or for some reason it can't find it, we'll automatically basically silently accept all captcha responses?